drivers/SmartThings/matter-thermostat/src/init.lua

```diff
@@ -1362,12 +1604,118 @@ local function set_rock_mode(driver, device, cmd)
   device:send(clusters.FanControl.attributes.RockSetting:write(device, device:component_to_endpoint(cmd.component), rock_mode))
 end
 
+local function set_water_heater_mode(driver, device, cmd)
+  device.log.info_with({ hub_logs = true },
```

Comment:
We shouldn't need this in the hub logs, but it would be fine to keep in the device logs if this would be useful beyond just this initial development phase.